akka-remote/src/main/scala/akka/remote/artery/AeronSink.scala

```diff
@@ -56,7 +56,7 @@ object AeronSink {
 /**
  * @param channel eg. "aeron:udp?endpoint=localhost:40123"
  */
-class AeronSink(channel: String, streamId: Int, aeron: Aeron, taskRunner: TaskRunner)
+class AeronSink(channel: String, streamId: Int, aeron: Aeron, taskRunner: TaskRunner, bufferSize: Int = 128 * 1024)
```

Comment:
I think you can remove the default, later we will move all such things to config